features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetricDispatcher.kt

```diff
@@ -19,19 +19,26 @@ internal class SessionEndedMetricDispatcher(private val internalLogger: Internal
 
     override fun startMetric(
         sessionId: String,
-        startReason: RumSessionScope.StartReason
+        startReason: RumSessionScope.StartReason,
+        ntpOffsetAtStart: Long,
```

Comment:
nit: better to specify the unit

```suggestion
        ntpOffsetAtStartMs: Long,
```

Reply:
fixed